migration/v1_to_v2/data/exporters/data_exporter.rb

```diff
@@ -148,13 +148,14 @@ def value_to_ruby_string(value, include_blank = false)
   # rubocop:enable Metrics/AbcSize
 
   def valid_key?(key)
-    return false if key.is_a?(Integer) || key.include?('-') || key.include?(' ') || !key.match?('^[a-zA-Z]')
+    return false if key.is_a?(Integer) || key.include?('-') || key.include?(' ') || !key.match?('^[a-zA-Z]') || key.include?("'")
 
     true
   end
 
   def key_to_ruby(key)
-    valid_key?(key) ? key : "'#{key}'"
+    # TODO: This doesn't handle the case where the key has both ' and " in it
+    valid_key?(key) ? key : (key.include?("'") ? "\"#{key}\"" : "'#{key}'")
```

Comment:
please check comment in the PR https://github.com/primeroIMS/primero-v2-migration/pull/4/files#diff-1b7e9813aba76827c12838aac2a3d3f30fc92590b9193b92c6dea3b86cfe726e